lib/api/client-commands/_base-command.ts

```diff
@@ -76,11 +101,14 @@ module.exports = class ClientCommand {
    * @param {function} callback
    * @private
    */
-  executeScriptHandler(method, script, args, callback) {
+
+  transportActions : any = {}
+
+  executeScriptHandler(method : string, script : string | Function, args : any[], callback : (param : any) => void) {
     let fn;
 
-    if (script.originalTarget) {
-      script = script.originalTarget;
+    if ((script as any).originalTarget) {
+      script = (script as any).originalTarget;
```

Comment:
We don't want to use `any` anywhere, unless absolutely necessary. So, we would need to make changes in the argument itself to accommodate `originalTarget` as well.

Ex. `script: string | Function | {originalTarget: string | Function}`

And then try to implement the DRY principal (`string | Function` are repeating twice above).